src/components/Change/CompletingParty.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <section id="completing-party-section">
+    <h2>{{sectionNumber}} Completing Party</h2>
+
+    <div class="mt-4" :class="{ 'invalid-section': invalidSection }">
+      <!-- FUTURE: update this component so it doesn't set flag until user changes something -->
+      <CompletingPartyShared
+        class="section-container py-6"
+        :completingParty="getCompletingParty"
+        :enableAddEdit="isRoleStaff"
+        :addressSchema="PersonAddressSchema"
+        :validate="validate"
+        :invalidSection="invalidSection"
+        @update="onUpdate($event)"
+        @valid="onValid($event)"
+      />
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { Action, Getter } from 'vuex-class'
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { ActionBindingIF } from '@/interfaces/'
+import { CompletingPartyIF } from '@bcrs-shared-components/interfaces/'
+import { CompletingParty as CompletingPartyShared } from '@bcrs-shared-components/completing-party/'
+import { PersonAddressSchema } from '@/schemas/'
+
+@Component({
+  components: {
+    CompletingPartyShared
+  }
+})
+export default class CompletingParty extends Vue {
```

Comment:
This is a "shim compoenent" that includes things the parent (Change page) doesn't need to know about and handles things (store getters/actions) that the shared sub-component can't access.